components/scroller/scroller.test.js

```diff
@@ -0,0 +1,87 @@
+import { assert } from 'chai';
+import { mount } from '@vue/test-utils';
+import DtScroller from './scroller.vue';
+
+global.requestAnimationFrame = cb => cb();
```

Comment:
Mhmm.. I saw we use 

`global.requestAnimationFrame = sinon.spy();`

But it not work... Will investigate this

Reply:
I updated it with 

`  before(function () {
    global.requestAnimationFrame = sinon.spy();
  });
`
But as you can see, Im not calling 
`global.cancelAnimationFrame = sinon.spy();`
and also 
`config.global.renderStubDefaultSlot = true;`

They are necessary to call? or it is okay on this way?

Reply:
unless you are getting errors when running the tests this is fine.